### PR TITLE
Updates Hardware LCM to support MySQL Compliant Queries

### PR DESCRIPTION
**Pull request by josh5276**
Updates Hardware LCM to support MySQL Compliant Queries (from Issue #28)
Also updates the form for hardware notices to have a similar look as the software form as to keep the plugin consistent.

### CODE REVIEW

**Review by dgarros — COMMENTED** [review]
This looks brilliant, well done @josh5276

File: development/Dockerfile [hunk 1]
```diff
@@ -8,6 +8,8 @@ ARG NAUTOBOT_ROOT=/opt/nautobot
 
 ENV NAUTOBOT_ROOT ${NAUTOBOT_ROOT}
 
+RUN apt-get update && apt-get install -y gcc libmariadb-dev-compat libmariadb-dev
```
Comment (nniehoff) [inline development/Dockerfile:11]: this seems like A LOT just for MySQL client support

**Comment by brobare** [comment]
@josh5276 Think this may be able to get merged soon?

**Comment by josh5276** [comment]
Apologies for the delay @brobare. I'll get this merged and draft a new release.

**Pull request merged**